apps/web/pages/settings/my-account/calendars.tsx

```diff
@@ -102,167 +46,9 @@ const CalendarsView = () => {
-                              </Link>
-                              {data?.destinationCalendar?.credentialId === item.credentialId && (
-                                <Badge variant="green">Default</Badge>
-                              )}
-                            </ListItemTitle>
-                            <ListItemText component="p">{item.integration.description}</ListItemText>
-                          </div>
-                          <div>
-                            <DisconnectIntegration
-                              trashIcon
-                              credentialId={item.credentialId}
-                              buttonProps={{ className: "border border-default" }}
-                            />
-                          </div>
-                        </div>
-                        <div className="border-subtle w-full border-t">
-                          <p className="text-subtle px-2 pt-4 text-sm">{t("toggle_calendars_conflict")}</p>
-                          <ul className="space-y-4 p-4">
-                            {item.calendars.map((cal) => (
-                              <CalendarSwitch
-                                key={cal.externalId}
-                                credentialId={cal.credentialId}
-                                externalId={cal.externalId}
-                                title={cal.name || "Nameless calendar"}
-                                name={cal.name || "Nameless calendar"}
-                                type={item.integration.type}
-                                isChecked={
-                                  cal.isSelected || cal.externalId === data?.destinationCalendar?.externalId
-                                }
-                              />
-                            ))}
-                          </ul>
-                        </div>
-                      </ListItem>
-                    )}
-                  </Fragment>
-                ))}
-              </List>
-            </div>
-          ) : (
-            <EmptyScreen
-              Icon={Calendar}
-              headline={t("no_calendar_installed")}
-              description={t("no_calendar_installed_description")}
-              buttonText={t("add_a_calendar")}
-              buttonOnClick={() => router.push("/apps/categories/calendar")}
-              className="mt-6"
-            />
-          );
-        }}
-        error={() => {
-          return (
-            <Alert
-              message={
-                <Trans i18nKey="fetching_calendars_error">
-                  An error ocurred while fetching your Calendars.
-                  <a className="cursor-pointer underline" onClick={() => query.refetch()}>
-                    try again
-                  </a>
-                  .
-                </Trans>
-              }
-              severity="error"
-            />
-          );
-        }}
-      />
+      <div className="mt-8">
+        <CalendarListContainer />
```

Comment:
NIT: Not blocking

Wrong skeleton

<img width="1508" alt="Screenshot 2023-11-17 at 11 24 48 PM" src="https://github.com/calcom/cal.com/assets/53316345/f2d5ebd2-a01d-456d-a623-82dfbff951d1">